### ISSUE TITLE
Document storage requirement

### ISSUE THREAD

**Issue by jsfan3** [mudler/LocalAI#2869]
I tried the Docker installation indicated on the home page:
```
docker run -p 8080:8080 --gpus all --name local-ai -ti localai/localai:latest-aio-gpu-nvidia-cuda-12
```
It fails because there is not enough free space in my VPS. Could you document for the installation options listed at https://localai.io/#start-localai what the disk space requirements are?

**Comment by neoacevedo** [comment]
I agree. Taking a little bit of calculation, for the AIO is around 40GB at least, but in RAM? What's the minimum?
